### PR TITLE
Update nft features doc

### PR DESCRIPTION
**Pull request by dgsbl**

### CODE REVIEW

**Review by SegueII — APPROVED** [review]

File: docs/features/nft.md [hunk 1]
```diff
@@ -10,7 +10,9 @@ nft metadata (`metadata`) can be stored directly on the chain, or the `URI` of i
 
 nft need to be issued before creation to declare their abstract properties:
 
--_Denom_: the globally unique asset class identifier
+-_Denom_: the globally unique asset class name
```
Comment (chengwenxi) [inline docs/features/nft.md:13]:
```suggestion
-_Denom_: the globally unique nft category name
```

File: docs/features/nft.md [hunk 2]
```diff
@@ -10,7 +10,9 @@ nft metadata (`metadata`) can be stored directly on the chain, or the `URI` of i
...
+
+-_Denom ID_: the globally unique asset class identifier of Denom
```
Comment (chengwenxi) [inline docs/features/nft.md:15]:
```suggestion
-_Denom ID_: the globally unique nft category identifier of Denom
```

File: docs/features/nft.md [hunk 3]
```diff
@@ -28,22 +30,22 @@ Each specific nft is described by the following elements:
 
 ### issued
 
-Specify the nft Denom (nft category) and metadata JSON Schema to issue nft.
+Specify the nft Denom (nft category), metadata JSON Schema, sender, chain ID and fee to issue nft.
```
Comment (chengwenxi) [inline docs/features/nft.md:33]: issue nft need additional fee? If it's the tx fees, delete it.

Comment (chengwenxi) [reply docs/features/nft.md:33]: delete chain-id

File: go.mod [hunk 4]
```diff
@@ -8,7 +8,7 @@ require (
 	github.com/golang/protobuf v1.4.3
 	github.com/gorilla/mux v1.8.0
 	github.com/grpc-ecosystem/grpc-gateway v1.16.0
-	github.com/irisnet/irismod v1.2.0
+	github.com/irisnet/irismod v1.2.1-0.20210119082202-5a59684d2b78
```
Comment (chengwenxi) [inline go.mod:11]: update irismod version via another PR

**Review by chengwenxi — CHANGES_REQUESTED** [review]
No need to specify global params in this doc.

**Review by chengwenxi — APPROVED** [review]

**Pull request merged**